code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -139,6 +142,11 @@
 	if(opened || !can_open(user))
 		return
 	playsound(loc, open_sound, open_sound_volume, TRUE, -3)
+	if(cooldown < world.time)
+		for(var/mob/living/carbon/human/H in contents)
+			if(prob(alt_open_sound_prob))
+				playsound(H.loc, 'sound/machines/closet_open_alt.ogg', open_sound_volume, FALSE)

```

Comment:
H.loc is src, doesn't matter since the first thing in playsound is get_turf anyways.